python/whylogs/core/constraints/metric_constraints.py

```diff
@@ -201,12 +327,11 @@ def validate(self, profile_view: Optional[DatasetProfileView] = None) -> bool:
                 if not result:
```

Comment:
Line 193/319 is a bug. `column_constraints` can be empty while `dataset_constraints` is not.